requirements.txt

```diff
@@ -0,0 +1,5 @@
+colour==0.1.5
```

Comment:
I think this should be colour-science!

Reply:
makes sense XD,  I'll close this PR and resubmit later on.